.github/workflows/test-spec.yml

```diff
@@ -28,38 +28,38 @@ jobs:
             admin_password: admin
           - version: 2.0.0
             admin_password: admin
-          - version: 2.18.0
-          - version: 2.18.0
+          - version: 2.19.0
+          - version: 2.19.0
             tests: plugins/index_state_management
-          - version: 2.18.0
+          - version: 2.19.0
             tests: plugins/ml
-          - version: 2.18.0
+          - version: 2.19.0
             tests: routing
-          - version: 2.18.0
+          - version: 2.19.0
             tests: snapshot
-          - version: 2.18.0
+          - version: 2.19.0
             tests: remote_store
-          - version: 2.18.0
+          - version: 2.19.0
             tests: dangling
             url: http://localhost:9200
-          - version: 2.18.0
+          - version: 2.19.0
             tests: plugins/replication
             url: http://localhost:9200
-          - version: 2.18.0
+          - version: 2.19.0
             tests: plugins/streaming
-          - version: 2.18.0
+          - version: 2.19.0
             tests: plugins/notifications
-          - version: 2.18.0
+          - version: 2.19.0
             tests: plugins/query_insights
-          - version: 2.18.0
+          - version: 2.19.0
             tests: plugins/workload-management
-          - version: 2.18.0
+          - version: 2.19.0
             tests: plugins/analysis
-          - version: 2.18.0
+          - version: 2.19.0
             tests: plugins/security
             cert: tests/plugins/security/.kirk.pem
             key: tests/plugins/security/.kirk-key.pem
-          - version: 2.19.0
+          - version: 2.20.0
             hub: opensearchstaging
             ref: '@sha256:4da23e0137b2b67206d23b36fcf0914cc39b3bf19310c782f536e4934b86f6cc'
```

Comment:
This is an old SHA from 2.19, if there's a 2.20 build, this SHA should be replaced from dockerhub.